sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -16589,7 +16589,7 @@
               "inputFormat": "plain",
               "validate": {
                 "required": true,
-                "custom": "",
+                "custom": "valid = (input >= 0 && input < 100000000) ? true : 'The number you have entered is not an acceptable value.'",
```

Comment:
I would recommend to create the max income/money value as hidden variable with value 10M to be re-used in many places.

Reply:
I believe you meant 100M.